translator/tests/acceptance/steps/actions.py

```diff
@@ -4,16 +4,18 @@
 from behave import then, when
 
 
-@when("we add {route} to the block list")
-def add_block(context, route):
+@when("we add {route} with {asn} and {community} to the block list")
+def add_block(context, route, asn, community):
     ip = ipaddress.ip_interface(route)
-    context.gobgp.add_path(ip)
+    event_data = {"asn": int(asn), "community": int(community)}
```

Comment:
I'm not sure these need to be ints? Something might just convert them.

Reply:
I was seeing failures because these were strings. I guess we could force the type elsewhere in the code instead of just here if that makes more sense?

Reply:
Ok, that's fine. At some point we should document what event_data is (i.e. "asn", "community", must be ints...).